SN2013ak's claimedtype from 2013ATel.4874....1M,Transient Name Server…

fedhere opened the pull request:
…,2000A&AS..143....9W marked as being erroneous.

This telegramby Milasalevich+ (2013ATel.4943....1M) classifying Sn 2013ak as IIb superseeds the earliermoregenericclassification as type II also by Milasalevich+ (2013ATel.4874....1M)

fedhere committed:
SN2013ak's claimedtype from 2013ATel.4874....1M,Transient Name Server,2000A&AS..143....9W marked as being erroneous.

This telegramby Milasalevich+ (2013ATel.4943....1M) classifying Sn 2013ak as IIb superseeds the earliermoregenericclassification as type II also by Milasalevich+ (2013ATel.4874....1M)

guillochon commented:
A couple small errors:
- There's a missing ] for "sources", which causes some syntax errors, which you can see highlighted in dark red [here](https://github.com/astrocatalogs/sne-internal/pull/28/files).
- The source should be "1", not "2" (there's only 1 source).

And a couple suggestions, which aren't required but might be helpful:
- If a publication has a bibcode, you only need to specify the bibcode and alias in the source. The rest of the fields will be generated automatically later. Less typing.
- If there's only one source, you don't need to define the alias, or attach a source to the quantities. Again less typing.
- GitHub by default uses spaces in its editor but the template uses tabs. If you change the editor tabbing from "spaces" to "tabs" it will make the whitespace more consistent.

fedhere commented:
ok, im catching up, but i submitted a few more before i saw this, which
probably have errors (the sourcenumber, which I was putting as it should
have been in chronological order in the end result) , and mistakes, like
the dumb missing ]

On Sun, Aug 28, 2016 at 7:14 PM, James Guillochon notifications@github.com
wrote:

> A couple small errors:
> - There's a missing ] for "sources", which causes some syntax errors,
>   which you can see highlighted in dark red here
>   https://github.com/astrocatalogs/sne-internal/pull/28/files.
> - The source should be "1", not "2" (there's only 1 source).
> 
> And a couple suggestions, which aren't required but might be helpful:
> - If a publication has a bibcode, you only need to specify the bibcode
>   and alias in the source. The rest of the fields will be generated
>   automatically later. Less typing.
> - If there's only one source, you don't need to define the alias, or
>   attach a source to the quantities. Again less typing.
> - GitHub by default uses spaces in its editor but the template uses
>   tabs. If you change the editor tabbing from "spaces" to "tabs" it will make
>   the whitespace more consistent.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/astrocatalogs/sne-internal/pull/28#issuecomment-243005586,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/ABnkhmVoOMF_i2BpzOox-XaMsArlwt7fks5qkhZEgaJpZM4JvCvJ
> .

## 

 /|/\
   |\*  Please consider the environment before printing this e-mail

guillochon commented:
No worries, take your time!

fedhere committed:
Update SN2013ak.json

fedhere committed:
Update SN2013ak.json

ok, i tried to fix the errors and mistakes in my earlier json commit according to our conversation by email.

guillochon commented:
Looks good now!
